bug in running the tool

When I run `npm run doc:generate` I got the following error:

```
[INFO] 12:27:51 ts-node-dev ver. 2.0.0 (using ts-node ver. 10.9.2, typescript ver. 5.3.3)
SyntaxError: /Users/batool/Desktop/GitHub/OSCSA/JSquarto/tsconfig.json: Expected property name or '}' in JSON at position 29
    at parse (<anonymous>)
    at Module._extensions..json (node:internal/modules/cjs/loader:1286:39)
    at Object.nodeDevHook [as .json] (/Users/batool/Desktop/GitHub/OSCSA/JSquarto/node_modules/ts-node-dev/lib/hook.js:63:13)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Function.Module._load (node:internal/modules/cjs/loader:922:12)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/batool/Desktop/GitHub/OSCSA/JSquarto/node_modules/ts-node-dev/lib/wrap.js:104:1)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._compile (/Users/batool/Desktop/GitHub/OSCSA/JSquarto/node_modules/source-map-support/source-map-support.js:568:25)
[ERROR] 12:27:51 SyntaxError: /Users/batool/Desktop/GitHub/OSCSA/JSquarto/tsconfig.json: Expected property name or '}' in JSON at position 29
```

Can you look into it @RealRichi3 ?

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "doc:generate": "ts-node-dev tsconfig.json ./src/index.ts"
+    "doc:generate": "ts-node-dev --project tsconfig.json ./src/index.ts"
   },
   "keywords": [],
   "author": "",

